tst/SOTGroupsInformation.tst

```diff
@@ -0,0 +1,248 @@
+#
+gap> START_TEST("SOTGroupsInformation.tst");
+
+#
+gap> SOTGroupsInformation(5);
+
+  There is 1 group of order 5.
+
+  There is 1 cyclic group.
+
+#
+gap> SOTGroupsInformation(5^2);
+
+  There are 2 groups of order 25.
+
+  There is 1 cyclic group, and 1 elementary abelian group.
+
+#
+gap> SOTGroupsInformation(5^3);
+
+  There are 5 groups of order 125.
+
+  There are 3 abelian groups, and 2 extraspecial groups.
+
+#
+gap> SOTGroupsInformation(5^4);
+
+  There are 15 groups of order 625.
+
+  There are 5 abelian groups, and 10 nonabelian groups.
+
+#
+gap> SOTGroupsInformation(5*3);
+
+  There is 1 group of order 15.
+
+  There is 1 cyclic group.
+
+#
+gap> SOTGroupsInformation(5*11);
+
+  There are 2 groups of order 55.
+
+  There is 1 cyclic group, and 1 nonabelian group.
+
+#
+gap> SOTGroupsInformation(5^2*3);
+
+  There are 3 groups of order 75.
+
+  There are 2 abelian groups.
+
+#
+gap> SOTGroupsInformation(5^3*3);
+
+  There are 7 groups of order 375.
+
+  The groups of order p^3q are solvable by Burnside's pq-Theorem.
+  These groups are sorted by their Sylow subgroups.
+    SOT 1 - 3 are abelian.
+    SOT 4 - 5 are nonabelian nilpotent and have a normal Sylow 
+    5-subgroup and a normal Sylow 3-subgroup.
+    SOT 6 is non-nilpotent and has a normal Sylow 3-subgroup with Sylow 
+    5-subgroup [ 125, 5 ].
+    SOT 7 is non-nilpotent and has a normal Sylow 3-subgroup with Sylow 
+    5-subgroup [ 125, 3 ].
+
+#
+gap> SOTGroupsInformation(5^4*3);
+
+  There are 21 groups of order 1875.
+
+  The groups of order p^4q are solvable by Burnside's pq-Theorem.
+  These groups are sorted by their Sylow subgroups.
+    SOT 1 - 15 are nilpotent and all Sylow subgroups are normal.
+    SOT 16 is sovable, non-nilpotent and has a normal abelian Sylow 
+    5-subgroup [ 625, 2 ], with cyclic Sylow 3-subgroup.
+    SOT 17 is sovable, non-nilpotent and has a normal abelian Sylow 
+    5-subgroup [ 625, 11 ], with cyclic Sylow 3-subgroup.
+    SOT 18 - 
+    19 are sovable, non-nilpotent and have a normal elementary abelian Sylow 
+    5-subgroup [ 625, 15 ], with cyclic Sylow 3-subgroup.
+    SOT 20 is sovable, non-nilpotent and has a normal nonabelian Sylow 
+    5-subgroup [ 625, 14 ], with cyclic Sylow 3-subgroup.
+    SOT 21 is sovable, non-nilpotent and has a normal nonabelian Sylow 
+    5-subgroup [ 625, 12 ], with cyclic Sylow 3-subgroup.
+
+#
+gap> SOTGroupsInformation(5^2*3^2);
+
+  There are 6 groups of order 225.
+
+  The groups of order p^2q^2 are solvable by Burnside's pq-Theorem.
+  These groups are sorted by their Sylow subgroups.
+    SOT 1 - 4 are abelian and all Sylow subgroups are normal.
+    SOT 5 is non-abelian, non-nilpotent and has a normal Sylow 5-subgroup 
+    [ 25, 2 ] with Sylow 3-subgroup [ 9, 1 ].
```

Comment:
... and it would be nice if this was indented, perhaps like so:
```suggestion
    SOT 5 is non-abelian, non-nilpotent and has a normal Sylow 5-subgroup 
          [ 25, 2 ] with Sylow 3-subgroup [ 9, 1 ].
```